packages/dd-trace/src/exporters/agent/writer.js

```diff
@@ -42,7 +42,7 @@ class Writer extends BaseWriter {
       startupLog({ agentError: err })
 
       if (err) {
-        log.error('Error sending payload to the agent (status code: %s)', err.status, err)
+        log.error('Error sending payload to the agent (status code: %s)', err.status, { transmit: false }, err)
```

Comment:
To reduce the risk of false positives, shouldn't we use a symbol?

```suggestion
        log.error('Error sending payload to the agent (status code: %s)', err.status, { [Symbol.for('datadog:transmit')]: false }, err)
```

Reply:
I suppose I could make the last argument _just_ a symbol (not an object with a symbol property). Then check if the last argument is a symbol and narrow it down to the transmit skip symbol.